altair/utils/html.py

```diff
@@ -293,6 +326,12 @@ def spec_to_html(
         vlc = import_vl_convert()
         vl_version = vl_version_for_vl_convert()
         render_kwargs["vegaembed_script"] = vlc.javascript_bundle(vl_version=vl_version)
+    elif template == "olli":
+        olli_version = "2"
+        olli_adapters_version = "2"
+        render_kwargs["olli_version"] = olli_version
+        render_kwargs["olli_adapters_version"] = olli_adapters_version
```

Comment:
```suggestion
        OLLI_VERSION = "2"
        OLLI_ADAPTERS_VERSION = "2"
        render_kwargs["olli_version"] = OLLI_VERSION
        render_kwargs["olli_adapters_version"] = OLLI_ADAPTERS_VERSION
```

Reply:
This isn't blocking for me btw, just thought it would help keep the constants easy to find if/when they need updating